src/types/onyx/OnyxCommon.ts

```diff
@@ -8,7 +9,9 @@ type PendingFields<TKey extends string = string> = Record<TKey, PendingAction |
 
 type ErrorFields<TKey extends string = string> = Record<TKey, Errors | null | undefined>;
 
-type Errors = Record<string, string | null>;
+type Error = string | MaybePhraseKey;
+
+type Errors = Record<string, Error | null>;
```

Comment:
What's the reason of this change? Let's be careful because `Errors` is a type used in many files

Reply:
Its because for example of this https://github.com/Expensify/App/blob/f697c897d05588f58aba426199f4cc08638711b3/src/pages/settings/Profile/PersonalDetails/LegalNamePage.js#L65-L67

here we are not adding just string but this `MaybePhraseKey`